.env

```diff
@@ -142,3 +149,7 @@ HEARING_POST_FORM_URL='https://os2forms.example.com/tvist1-borger?case-id=%CASE_
 # Text to show on hearing post form link (will not be translated). If empty, the
 # text "Open hearing post form" (with translation applied) will be used.
 HEARING_POST_FORM_LINK_TEXT=''
+
+# OS2Forms required tokens
+TVIST1_API_TOKEN=''
+SELVBETJENING_USER_API_TOKEN=''
```

Comment:
Why is this called `SELVBETJENING_USER_API_TOKEN` and not just `SELVBETJENING_API_TOKEN`?

Reply:
To authenticate ourselves in `selvbetjening`, when calling the REST API endpoints, we need the API token from a user that has been granted API access to the respective forms.